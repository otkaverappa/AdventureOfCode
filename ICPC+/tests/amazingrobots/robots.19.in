20 20
####################
#X#...#...#...#....#
#.#.#.#.#.#.#.#.#.##
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.##.#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#.##
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.##.#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#.##
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#...##.#
#.#.#.#.#.#.#####..#
#.#.#.#.#.#.....#.##
#.#.#.#.#.#####.#..#
#...#...#.......#.##
#################.##
1
18 18 2 E
20 20
#.##################
#.#....#...#...#...#
#....#...#...#...#.#
##################.#
#..................#
#.##################
#..................#
##################.#
#..................#
#.##################
#..................#
##################.#
#..................#
#.##################
#..................#
##################.#
#..................#
#.##################
#.................X#
####################
1
3 4 2 E
