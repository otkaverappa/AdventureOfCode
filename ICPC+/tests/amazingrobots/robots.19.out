314
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
N
N
E
E
E
E
E
E
E
E
E
E
E
E
E
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
N
N
E
E
E
E
E
E
E
E
E
E
E
E
E
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
N
N
E
E
E
E
E
E
E
E
E
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
E
E
E
E
N
N
W
W
W
W
N
N
N
N
N
N
N
N
N
N
N
N
N
W
W
W
W
W
W
W
W
W
W
W
W
W
N
N
E
E
E
E
E
E
E
E
E
E
E
E
E
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
E
E
E
W
S
W
S
E
S
W
S
W
N
W
W
S
S
E
S
W
S
W
N
W
W
S
S
E
S
S
S
W
S
W
N
W
W
W
S
W
S
S
W
S
S
N
N
N
